.gitignore

```diff
@@ -8,3 +8,4 @@
 /laminas-mkdoc-theme/
 /phpunit.xml
 /vendor/
+/.idea
```

Comment:
This should be added to your personal `~/.gitignore_global`

Reply:
See also in the documentation of Git: https://git-scm.com/book/en/v2/Customizing-Git-Git-Configuration